### DISCUSSION TITLE
P-listbox Allow to disable individual item

### DISCUSSION THREAD

**Discussion by siva-geddada** [primefaces/community#625] [PrimeNG] [answered]
Currently i  am using p-listbox . I had scenario like. I have to show 3 list-boxes and it having same data . If i select 1st value in 1st  listbox that 1st item should be disable in another list boxes . There are [disabled] [readonly] properties  but these or not value based and index based.
If i use any one of the property. it disables all the items in listbox

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback, we've added it in our internal backlog.
